skopt/utils.py

```diff
@@ -222,6 +222,7 @@ def expected_minimum(res, n_random_starts=20, random_state=None):
     """
     def func(x):
         reg = res.models[-1]
+        x = res.space.transform(x.reshape(1, -1))
```

Comment:
We forgot to transform the values that `sp_minimize` passes in. An alternative would be to run the minimizer on the transformed space/bounds (as we do internally). This is the 'minimal' fix though.

Reply:
Opinions @glouppe or should we even move this to a separate PR?

Reply:
It is good like this.

Reply:
I think this change warrants a new pip release.